Apps/GatewayApi/src/appsettings.hgpoc.json

```diff
@@ -40,9 +40,7 @@
         "TimeOuts": {
             "ResendSMS": "1"
         },
-        "Modules": {
-            "WalletExport": true
```

Comment:
Isn't this used for the mobile app?

Reply:
For the BC Wallet app, not our mobile app.